src/functionBased/components/ToDoContainer.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from 'react';
+import { v4 as uuid } from 'uuid';
+
+import { Route, Switch } from 'react-router-dom';
+import ToDoList from './ToDoList';
+import Header from './Header';
+import InputToDo from './InputToDo';
+import '../App.css';
+import About from '../pages/About';
+import NotMatch from '../pages/NotMatch';
+
+const TodoContainer = () => {
+  function getInitialTodos() {
+    return JSON.parse(localStorage.getItem('todos')) || [];
+  }
+  const [todos, setTodos] = useState(getInitialTodos);
+
+  useEffect(() => {
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }, [todos]);
+  const handleChange = (id) => {
+    setTodos(todos.map((todo) => {
+      const currentTask = todo;
+      if (currentTask.id === id) { currentTask.completed = !currentTask.completed; }
+      return currentTask;
+    }));
+  };
+  const delTask = (id) => {
+    setTodos((oldState) => (
+      oldState.filter((todo) => todo.id !== id)
+    ));
+  };
+  const addTask = (title) => {
+    setTodos((oldState) => ([
+      ...oldState,
+      {
+        id: uuid(),
+        title,
+        completed: false,
+      }]
+    ));
+  };
+  const setUpdate = (updatedTitle, id) => {
+    setTodos(todos.map((todo) => {
+      const currentTask = todo;
+      if (currentTask.id === id) { currentTask.title = updatedTitle; }
+      return currentTask;
+    }));
+  };
+
+  return (
+    //  <Routes>
+    //     <Route path="/" element={} />
+    //     <Route path="/about" element={<About index />} />
+    //     <Route path="*" element={<NotMatch />} />
+    //   </Routes>
```

Comment:
Kindly remove the comments as they are no longer necessary.

Reply:
### @jsug9 Nice catch 🙌. Will do.😎